### PR TITLE
Improved version of PR 82

### PR DESCRIPTION
**Pull request by zerc**
Improved version of #82

### CODE REVIEW

File: tests/functional/test_models.py [hunk 1]
```diff
@@ -82,6 +87,18 @@ class Employee(models.Model):
         emp = Employee(cv='3addab78-6368-4c55-ac08-22412b6a2a4c')
         self.assertIsInstance(emp.cv, File)
 
+    @patch('pyuploadcare.api_resources.rest_request', autospec=True)
+    def test_not_found_file_uuid(self, request):
+        request.side_effect = InvalidRequestError(404)
+
+        class Employee(models.Model):
+            cv = FileField(validators=[faked_size_validator])
```
Comment (homm) [inline tests/functional/test_models.py]: It is better to place the validator inside the `test_not_found_file_uuid` until it used somewhere else.

**Comment by homm** [comment]
Tests in Django < 1.8 failed.

**Comment by zerc** [comment]
Working on it :construction_worker:

**Comment by zerc** [comment]
@homm  

I solved problems which happens when we use `mock.patch` and `db.Model` in a single file.

All tests pass.

File: tests/functional/test_models.py [hunk 2]
```diff
@@ -5,13 +5,26 @@
...
+# Add project folder to PATH for avoiding troubles in Django < 1.8
+# It can't find `gallery` app
+sys.path.append('test_project')
 os.environ['DJANGO_SETTINGS_MODULE'] = 'test_project.settings'
 
+# Load models for avoiding 'AppRegistryNotReady: Models aren't loaded yet.'
+# in Django 1.7.10
+import django
+if hasattr(django, 'setup'):
+    django.setup()
```
Comment (homm) [inline tests/functional/test_models.py]: All of this changes look unrelated to the new test. Why all this things happened now?

**Comment by homm** [comment]
Ok, I see what something wrong with tests launching, but fixing takes too many time.

:ship:

**Comment by hheimbuerger** [comment]
It's now no longer crashing and instead showing the message

> The file could not be found in your Uploadcare project

I assume that's what's supposed to happen now and better than before. I guess—together with the recent fix in the widget, that you can now remove an image if it's not available—this is pretty functional for my use case of importing a database full of references to another uploadcare project.

Thanks!

**Comment by dmitry-mukhin** [comment]
as soon as we can, we're overloaded atm :)
